arbor/backends/gpu/fvm.hpp

```diff
@@ -50,14 +50,16 @@ struct backend {
     using ion_state = arb::gpu::ion_state;
 
     static threshold_watcher voltage_watcher(
-        const shared_state& state,
+        shared_state& state,
```

Comment:
If we pass the `time_since_spike` vector as a non-const pointer to the `test` method, we don't need to store it in the `threshold_watcher`, so that fixes the problem.